analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/RestrictDeserializedTypes.cs

```diff
@@ -155,15 +156,15 @@ internal void Switch(Stream stream, bool condition, int number)
         switch (number)
         {
             case 1:
-                formatter4.Deserialize(stream);                                 // FIXME Non-compliant: null binder
+                formatter4.Deserialize(stream);                                 // Noncompliant: null binder
                 break;
             case 2:
                 formatter4.Binder = null;
-                formatter4.Deserialize(stream);                                 // FIXME Non-compliant: null binder
+                formatter4.Deserialize(stream);                                 // Noncompliant [nullBinder1]: null binder
```

Comment:
Can the secondary FP be fixed by removing the secondary location from the `Dictionary` when `null` (or something with `NullConstraint`) is assigned to the `Binder` property?

Reply:
As discussed, this already happens and does not fix the issue. The FP will be fixed when we implement flow tracking.